app/src/main/res/layout/fragment_ranking.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <TextView
+        android:layout_width="250dp"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:text="Congratulations to NICOLAS MATEKALO, OUR BEST USER" />
```

Comment:
I would put something like "NICOLAS MATEKALO biggest Golem at EPFL"
Also you forgot to put the name of the highest karma user (instead of a hardcoded string)

Reply:
Hahahaha, that's my bad, I was trying to display the best user name. But finally only the podium is displayed. So we should delete this line.